script/scan

```diff
@@ -21,7 +21,7 @@ pp proofer_opts
 puts
 
 begin
-  HTML::Proofer.new(File.join("tmp", "site"), proofer_opts).run
+  HTMLProofer.check_directory(File.join("tmp", "site"), proofer_opts).run
```

Comment:
seems so if I refer to https://github.com/gjtorikian/html-proofer/blob/master/Rakefile#L21